doc/source/getting_started/Introduction.ipynb

```diff
@@ -24,6 +24,13 @@
     "import numpy as np"
```

Comment:
<p>Here I would use </p><p> </p><pre class="ql-syntax" spellcheck="false">{doc}<code>examples/styles</code>
</pre><p> </p><p>There is also the option to use </p><p> </p><pre class="ql-syntax" spellcheck="false">{ref}<code>mpl_styles</code>
</pre><p> </p><p>which would also work, but I think we should update the gallery generator to change that before using it. mpl and bokeh used to be on different files but are not tabs within the same file, so now we don't need a tag for each file, only a tag for the example. If you prefer to use ref instead of doc, you can change the line https://github.com/arviz-devs/arviz/blob/main/doc/sphinxext/gallery_generator.py#L283 for </p><p> </p><pre class="ql-syntax" spellcheck="false">return f"example_{self.basename}"
</pre><p> </p><p>so that you can then use</p><p> </p><pre class="ql-syntax" spellcheck="false">{ref}<code>example_styles</code>
</pre>
<br/>

 _Reply via <a href='https://app.reviewnb.com/arviz-devs/arviz/pull/1779/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>

Reply:
In place of "examples of style sheets [here](https://arviz-devs.github.io/arviz/examples/styles.html)" ?

Reply:
yes, you can still use custom text for the links even if using sphinx cross references instead of html hyperlinks. i.e.

```
examples of style sheets {doc}`examples/styles`
```

Will use as text for the hyperlink the title of the linked page, in this case "Matplotlib styles" (https://arviz-devs.github.io/arviz/examples/styles.html). Whereas

```
examples of style sheets {doc}`here <examples/styles>`
```

will show "here" as text.